code/index.html

```diff
@@ -1,17 +1,77 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Business Site</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <h1>Business name 🌻</h1>
+<html lang="en">
```

Comment:
HTML: The code is well structured and clearly classes clearly labeled. For me as a reader not familiar with your code is it easier to have comments showing me where the different code sections start/end. 
Was there a reason why you did not include the hero image into your header section?